pycqed/instrument_drivers/physical_instruments/QuTech_CCL.py

```diff
@@ -95,15 +95,7 @@ def _init_submodules(self):
         curdir = os.path.dirname(__file__)
         print(f"Assembler version {self.QISA.getVersion()}")
         if self.QISA.getVersion() == '4.0.0':
```

Comment:
Looking at this, it is probably better to do: `if self.QISA.getVersion() != '2. ...':`.